src/main/java/org/mule/extension/email/api/exception/EmailCountMessagesException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.email.api.exception;
+
+import static org.mule.extension.email.internal.errors.EmailError.EMAIL_COUNT_MESSAGES;
```

Comment:
Please, remember to add this new error in the troubleshooting guide section: https://docs.mulesoft.com/email-connector/1.6/email-troubleshooting#understand-common-throws